src/internet_identity/internet_identity.did

```diff
@@ -357,6 +357,21 @@ type IdentityInfo = record {
     metadata: MetadataMap;
 };
 
+type CaptchaCreateResponse = variant {
+    ok: Challenge;
+};
+
+type IdentityRegisterResponse = variant {
+    // Registration successful.
+    ok: IdentityNumber;
+    // No more registrations are possible in this instance of the II service canister.
+    canister_full;
+    // The challenge was not successful.
+    bad_challenge;
```

Comment:
nit: this is slightly ambiguous, maybe using "captcha" in the name/comment would be better, e.g. 
```
// The provided CAPTCHA solution was incorrect
bad_captcha_solution;
```
I'm aware that "challenge" is used throughout the code, and e.g.  we have `ChallengeResult`-type rather than something like `CaptchaSolution` but maybe that's something we could "fix" in the future, and use more descriptive types/labels in the new API?  Or maybe it is not worth the effort?

Reply:
Ah, good point. 👍  I'll change it in the API v2 error so that we are consistently using `captcha` in that version.